core/boxes_manager.rb

```diff
@@ -42,4 +42,23 @@ def getBox(key)
     @boxesManager[key]
   end
 
+  def getBoxByConfig(config_path, node_name)
+    config = nil
+    begin
+      config = JSON.parse(File.read(config_path))
+    rescue
+      $out.error "Wrong config path or json implementation for #{config_path}"
```

Comment:
Such constructions when we return nil and print error to $out should be replaced for exception throwing.